### ISSUE TITLE
Add the ability to reorder the sidebar tabs

### ISSUE THREAD

**Issue by rune3132** [runelite/runelite#783]
I feel like the logout tab should be at the bottom, other people might want another order. Being able to reorder it would be nice.

**Comment by ShaunDreclin** [comment]
duplicate of #582

**Comment by deathbeam** [comment]
As Shaun said

**Comment by rune3132** [comment]
From my understanding, that issue is requesting the reordering of the ingame tiles. I would like to reorder the clients tabs, like the Xp Tracker, Info, Kurend Library plugin and so on.

**Comment by deathbeam** [comment]
Ah, yes, sorry, misunderstood the issue. Calling button tab is very confusing :( Anyway, for the login/logout problem, there is #646 PR open currently.

**Comment by ShaunDreclin** [comment]
Ohh I misunderstood too, yeah being able to re-order the client's tab strip would be nice.

Possibly related to that, #449

**Comment by rune3132** [comment]
No worries, i could have been more precise when describing my request. Fel free to reword anythign if there is some kind of standart that i didn't follow :)

**Comment by yoghurtis** [comment]
Yes please!

**Comment by RickertdeH** [comment]
This would be useful, especially with all the new external plugins available.

**Comment by wigglepop** [comment]
![image](https://user-images.githubusercontent.com/77034846/103752787-1e861000-5045-11eb-8ed3-8466aa8ca986.png)
if the ability to re-arrange sidebars is not going to be put, please add a scrolling ability for those who don't play full screen. it's hard to put the client to full screen again and again just so we could open up a plugin that's at the bottom of the list

**Comment by jevonsp** [comment]
Is this ever going to be revisited? with the plugin hub it is a larger annoyance

**Comment by mooseweener** [comment]
Agreed, this needs to be revisited. I play full screen and I just hit the max icons.

**Comment by jonathanjosephduarte** [comment]
scrolling or the ability to rearrange the icons would be excellent!
